### ISSUE TITLE
code for loss function L_{MI} and L_{ML}

### ISSUE THREAD

**Issue by tanshuai0219** [X-LANCE/AniTalker#4]
Will you release the code for loss function L_{MI} and L_{ML}, which are the core of this paper, please?

**Comment by liutaocode** [comment]
I'm very sorry that due to our policy, we are unable to provide this at the current stage. However, I can give you some ideas for reproduction. You can refer to the code below.

`
self.tc_loss = TCLineEstimator(
    dims=[style_dim,style_dim], 
    hidden_size=style_dim, 
    mi_estimator='CLUB'
)

tc_loss = self.tc_loss.learning_loss([feature_after_motion_encoder, feature_after_identity_encoder]) 
`
TCLineEstimator can be found [here](https://github.com/Linear95/TC-estimation/blob/master/tc_estimators.py#L16).
